src/components/messenger/list/conversation-item/index.tsx

```diff
@@ -58,23 +58,23 @@ export class ConversationItem extends React.Component<Properties> {
   };
 
   renderAvatar() {
-    if (this.props.conversation.isOneOnOne && this.props.conversation.otherMembers[0]?.profileImage) {
+    if (isCustomIcon(this.props.conversation.icon)) {
```

Comment:
Looks like we have 3 identical renderings of the Avatar component aside from the url.... bonus points to refactor this to only do it once. lol

Reply:
done, thanksss